runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -176,6 +176,13 @@ fn main() {
         }
     );
 
+    if let Some(outcome) = outcome.clone() {
+        println!("\nLogs:");
+        for log in outcome.logs {
+            println!("{}\n", log);
+        }
+    }
```

Comment:
```suggestion
    if let Some(outcome) = &outcome {
        println!("\nLogs:");
        for log in &outcome.logs {
            println!("{}", log);
        }
    }
```

Reply:
The `&` sounds good. but i don't suggest to remove `\n` because people always log without explicit `\n` (I don't see an exception, because logs are stored as elements in vec and rendered as different lines on explorer):
```
        env::log(
            format!(
                "@{} deposited {}. New unstaked balance is {}",
                account_id, amount, account.unstaked
            )
            .as_bytes(),
        );
```
So I think newline after each log here will make it readable and matches explorer displays.